shared/js/ui/templates/shared/normalize-company-icon-name.es6.js

```diff
@@ -0,0 +1,8 @@
+
+const bel = require('bel')
+
+module.exports = function (companyName) {
+  companyName = companyName || ''
+  const normalizedName = companyName.toLowerCase().replace(/\.[a-z]+$/, '')
+  return `${normalizedName}`
+}
```

Comment:
I'm worried we've been pushing logic into the templates. How about putting this in the top blocked model, where we build the list of companies in the first place?

I also think this needs to be tested. We can now add tests to UI models - hit me up if you want more info!